CHANGELOG.md

```diff
@@ -33,14 +38,11 @@ view-grid, view-list, virus, whatsapp, wheelchair-no, work
 
 Some icons have been updated, using the same machine name.
 
-### Image gallery multiple organism
-```
-<span>Show all photos <i class="icon-arrow-right" aria-hidden="true"></i></span>
-```
-Becomes
-```
-<span>Show all photos <i class="icon-fullscreen-enter" aria-hidden="true"></i></span>
-```
+### Heading atoms
+**Heading 2:** now has the blue (color-primary) color
```

Comment:
Dit hoeft niet noodzakelijk in de changelog. We voegen momenteel enkel noodzakelijke changes toe waar andere projecten kunnen naar kijken op de nieuwe stijl te implementeren, voornamelijk de HTML markup changes.